Common/Product/SharedProject/Automation/OAFileItem.cs

```diff
@@ -230,16 +267,13 @@ internal OAFileItem(OAProject project, FileNode node)
         /// <summary>
         /// Gets the ProjectItems for the object.
         /// </summary>
-        public override ProjectItems ProjectItems {
-            get {
-                return Node.ProjectMgr.Site.GetUIThread().Invoke<ProjectItems>(() => {
-                    if (this.Project.ProjectNode.CanFileNodesHaveChilds)
-                        return new OAProjectItems(this.Project, this.Node);
-                    else
-                        return base.ProjectItems;
-                });
-            }
-        }
+        public override ProjectItems ProjectItems => this.Node.ProjectMgr.Site.GetUIThread().Invoke<ProjectItems>(() =>
+                                                                   {
```

Comment:
this happens in a number of other files that have properties with nested lambdas.